src/components/Education.tsx

```diff
@@ -153,7 +154,7 @@ const Education = () => {
     const handleSave = () => {
     
         axios
-            .post("http://localhost:8081/education", {
+            .post(url + "/education", {
```

Comment:
@ddinh please store url + "/education" to a variable and pass here to minimize code duplication